### PR TITLE
Add Activeloop

### LINKED ISSUE #11088: Request: Deep Lake
### Brand Name

Deep Lake

### Website

https://www.deeplake.ai/

### Popularity Metric

Deep Lake is a multi-modal database for AI. It can store all types of data (vectors, images, texts, videos etc.). It can store, query, version & visualize any AI data. It is used to build data-driven applications with LLMs (Large Language Models). It is ranked 4,446,057 in SimilarWeb (Similarweb rank of its parent company [Activeloop](https://www.activeloop.ai/) is 522,074). Its [GitHub repo](https://github.com/activeloopai/deeplake) has 7.8k stars. The star history is on a consistently upward trajectory.

### Official Resources for Icon and Color

svg: https://docs.activeloop.ai/~gitbook/image?url=https%3A%2F%2F2322532613-files.gitbook.io%2F%7E%2Ffiles%2Fv0%2Fb%2Fgitbook-x-prod.appspot.com%2Fo%2Fspaces%252F-M_MXHpa1Cq7qojD2u_r%252Flogo%252FwpdFZDSecvZH3HgMaiT3%252Fdeeplake%2520logo.svg%3Falt%3Dmedia%26token%3D0ec4b846-b81b-4f31-b138-25f33dcbd90f&width=192&dpr=4&quality=100&sign=14e45f2dcd6c4bb4d77a40952faa99b12b035a9ee2f55f2687d488eb9321b3ab

File is uploaded below:
deeplake_logo

### Additional Comments

Deep Lake is the prominent product of Activeloop. Even though Deep Lake website has a Similarweb rank of 4,446,057, Activeloop website has a Similarweb rank of 522,074.

### LINKED ISSUE #11387: Request: Activeloop
### Brand Name

Activeloop

### Website

https://www.activeloop.ai/

### Popularity Metric

Activeloop provides Deep Lake - an Enterprise-Grade Database for AI. Deep Lake is a multi-modal database for AI. It can store all types of data (vectors, images, texts, videos etc.). It can store, query, version & visualize any AI data. It is used to build data-driven applications with LLMs (Large Language Models), LangChain etc. It can stream data in real-time to PyTorch or TensorFlow.

It is ranked 546,640 in SimilarWeb. Its [GitHub repo](https://github.com/activeloopai/deeplake) has 7.9k stars. The star history is on a consistently upward trajectory.

### Official Resources for Icon and Color

svg: https://www.activeloop.ai/static/logo-9fa16b423d87d3f5f877149da3ab5c9e.svg

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by prem-k-r**
image

**Issue:** closes #11387 & closes #11088

**Popularity metric:**

Similarweb: [Global 442k](https://www.similarweb.com/website/activeloop.ai/#overview)
Github: [8.7k](https://github.com/activeloopai/deeplake)

### Checklist

- [x] I have reviewed the [forbidden brands](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#forbidden-brands) list and confirm the brand I am submitting a PR for is not one of them, nor is it a subsidiary of one of those brands
- [x] I have reviewed the brand's terms of service, and am confident we can add this icon
- [x] I updated the JSON data in `data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

They got beautiful icon but the files are really bad and inconsistent
Took quite some efforts

color from [stylesheet](https://www.activeloop.ai/), variable named ~`brand`~ `hero`

### CODE REVIEW

**Comment by PeterShaggyNoble** [comment]
I'm going to ping @mikayelh for a review of this before we do ours to make sure they're OK with everything here:
- Are you happy with the monochrome treatment?
- Is that the correct colour for your brand?
- Are we right to be including Deep Lake as an alias, or is there another icons we should be using to represent it?
  - If we are right to be using it as an alias, is the Activeloop brand colour also the brand colour for Deep Lake, or should we use a different colour?
- Are there any brand guidelines or licensing information you would like us to include?

**Comment by mikayelh** [comment]
Hi @PeterShaggyNoble, @prem-k-r! thank you so so much for taking this on. 

A couple of notes - we're Activeloop, not ActiveLoop, in case this is propagated somewhere I don't see. Secondly, I'm looping in @davidbuniat on #1 (monochrome treatment). Here's the brandbook with more [info](https://www.figma.com/design/BkHRpmCiMOqY1iJMniKC6F/Activeloop-Brandbook--final-?node-id=3-384&t=D5cktutbqTdUM9rE-0).

The correct monochrome color is Hex: # FFB746, black looks good!

For the rest of the questions, I will defer to @davidbuniat since I'm no longer affiliated with the project/company.

**Comment by davidbuniat** [comment]
This looks great, thanks @mikayelh @PeterShaggyNoble @prem-k-r!

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
Thanks for the feedback, @mikayelh & @davidbuniat, much appreciated.

Before we merge this, though @prem-k-r, could you see if any of the excess points can be removed? If they can't then I'm happy to merge this as-is based on the approval of an official representative.

Also the alias should be a `dup` one. And there's an errant querystring at the end of the `guidelines` URL that should be removed.

**Comment by prem-k-r** [comment]
> Before we merge this, though @prem-k-r, could you see if any of the excess points can be removed? If they can't then I'm happy to merge this as-is based on the approval of an official representative.

I'll remove some more, please wait

> Also the alias should be a `dup` one. And there's an errant querystring at the end of the `guidelines` URL that should be removed.

I was also thinking dup would be better.
the parameter `?node-id=3-384` when removed it gives me first page only, so I left whole

**Comment by PeterShaggyNoble** [comment]
> the parameter ?node-id=3-384 when removed it gives me first page only

Ah, I see. In that case, then, we should change the `guidelines` to https://www.figma.com/design/BkHRpmCiMOqY1iJMniKC6F/Activeloop-Brandbook--final-?node-id=3-91
